### ISSUE TITLE
Removing a user from the queue does not remove them from the Round Robin

### ISSUE THREAD

**Issue by instaboostmedia** [mshanemc/roundRobinProcessBuilder#5]

**Comment by mshanemc** [comment]
Removing them from the public group, you mean?

**Comment by instaboostmedia** [comment]
Hmm that might be the problem lol. Let me try again and keep you posted.

**Comment by instaboostmedia** [comment]
> Removing them from the public group, you mean?

Apologies, it is the public group. Thank you for the quick response. By the way, this add on is the best thing we've installed!
